### ISSUE TITLE
Cannot load scss files, using vite.js in react 18.0 [a11y]: 

### ISSUE THREAD

**Issue by vitalijalbu** [carbon-design-system/carbon#11256]
### Package

@carbon/react, carbon-components, carbon-components-react

### Browser

Chrome

### Operating System

MacOS

### Package version

last

### React version

18

### Automated testing tool and ruleset

Codesandbox

### Description

Hi, i just installed carbon looking from the tutorial page: https://carbondesignsystem.com/developing/react-tutorial/step-1

But my react app doesn't laod .scss files.

### CodeSandbox example

https://codesandbox.io/s/blazing-dew-lwqv2i?file=/index.js

### Steps to reproduce

Just run it

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by vitalijalbu** [comment]
sorry, this is the codesandbox link: https://codesandbox.io/s/carbo-11-8v42uu

**Comment by joshblack** [comment]
Hi there @vitalijalbu! 👋 I went ahead and made a PR for an example with Vite: https://github.com/carbon-design-system/carbon/pull/11286

From what you shared, I think you'll run into some challenges with the following:

- Carbon doesn't currently support React 18 (but we will soon!)
- The styles are coming from `carbon-components` on v10 instead of v11, this is one of the c changes in this release where packages actually include the styles

In the PR, I used the `@carbon/react` package specifically for React components and styles.

Hope this helps! Let me know if you have any questions
